StRoot/StBFChain/BigFullChain.h

```diff
@@ -1587,6 +1587,8 @@ Bfc_st BFC[] = { // standard chains
                                                                                                     kFALSE},
   {"fstHit",     "", "fstChain", "event,fstCluster", "StFstHitMaker",       "StFstHitMaker","FST Hit Maker",
                                                                                                     kFALSE},
+  {"fstEvtRawHit", "", "", "fstRawHit",                       "","","Store FST Raw Hit in StEvent", kFALSE},
+  {"fstMuRawHit", "", "", "CMuDST",                             "","","Store FST Raw Hit in MuDst", kFALSE},
```

Comment:
I would assume (hope) that MuDst maker is reading FST raw hits from StEvent (our event data model) rather than a direct read from elsewhere.  So the fstMuRawHit option _implies_ the fstEvtRawHit option.  If so, I suggest...

` {"fstMuRawHit", "", "", "CMuDST,fstEvtRawHit",                             "","","Store FST Raw Hit in MuDst", kFALSE},`

If not... I may have other comments.

Reply:
Thanks for the comments. I agree with you that MuDst should be a subset of StEvent. This is the part I forgot to change when I include FST raw hits in StEvent. It is now implemented.